lib/DBDefs/Default.pm

```diff
@@ -136,6 +136,9 @@ sub CANONICAL_SERVER          { "https://musicbrainz.org" }
 # The server used to link to CritiqueBrainz users and reviews.
 sub CRITIQUEBRAINZ_SERVER     { "https://critiquebrainz.org" }
 
+# The URL where static resources are located, excluding the trailing slash.
+sub STATIC_RESOURCES_LOCATION { 'http://localhost:5000/static' }
```

Comment:
I think https://github.com/metabrainz/musicbrainz-server/commit/6d23419b5703d2df7c4d3d63d3f17931b0efd3fe fixes this, I don't see any immediate issues with JS, CSS, or images.

A sample of rev-manifest.json might look like this:
```
{"/build/jed-nl.js":"jed-nl-84597edbd4.js","/images/image404-125.png":"image404-125-98642f1825.png"}
```

The idea I went with is that the keys are the original paths (relative to static/) and the values are the filenames relative to `STATIC_RESOURCES_LOCATION`. Since everything is contained in build/, `STATIC_RESOURCES_LOCATION` should include build/.

It's a bit weird though, since the JS and CS don't have "original paths," they consist of many original files bundled together.